plugins/processors/execd/README.md

```diff
@@ -0,0 +1,73 @@
+# Execd Processor Plugin
+
+The `execd` processor plugin runs an external program as a separate process and pipes metrics in to the process's STDIN and reads processed metrics from its STDOUT. 
+The programs must output metrics in any one of the accepted 
+[Processor Data Formats](https://github.com/influxdata/telegraf/blob/master/docs/DATA_FORMATS_INPUT.md) on its standard output.
+
+The `signal` can be configured to send a signal the running daemon on each collection interval.
```

Comment:
We aren't supporting this right?  Also in the examples below and in the examples directory.